airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlCdcHelper.java

```diff
@@ -138,4 +139,26 @@ private static String getSchema(final ConfiguredAirbyteCatalog catalog) {
         .collect(Collectors.joining(","));
   }
 
+  private static String quoteIfNeeded(String input) {
```

Comment:
Will this work for a table name containing unicode characters?
can't we just quote everything?